dobc-centos/epel.repo

```diff
@@ -0,0 +1,8 @@
+[epel]
+name=Extra Packages for Enterprise Linux 7 - $basearch
+baseurl=http://epel.osuosl.org/7/$basearch
+enabled=1
+failovermethod=priority
+fastestmirror_enabled=0
```

Comment:
It looks like this is being ignored? I'm getting messages about centos trying to determine the fastest mirror when building images.

Reply:
There might be other repos that have it enabled. It shouldn't be a big deal.